data/changelog/ocaml/2023-08-??-ocaml-5.1.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Release of OCaml 5.1.0
+description: Release of OCaml 5.1.0
+date: "2023-08-??"
+tags: [ocaml, release]
+changelog: |
+---
+
+We have the pleasure of celebrating the birthday of ??? by announcing the release of OCaml version 5.1.0.
+
+Some of the highlights in OCaml 5.1.0 are:
+
+   - Many runtime performance regression fixes (dynlinking, weak array, weak hash sets, GC with idle domains, GC prefetching).
```

Comment:
GC with idle domains is more a memory-leak than a performance consideration.
```suggestion
   - Many runtime performance regression and memory-leaks fixes (dynlinking, weak array, weak hash sets, GC with idle domains, GC prefetching).
```

Reply:
Fixed, thanks !